### ISSUE TITLE
html file doesn't load

### ISSUE THREAD

**Issue by bvernot** [aaronmck/HW1_Aaron_McKenna#1]
The html file associated with this homework doesn't load, and there isn't a ".md" file to go with the .Rmd.  I'm not sure why this is.. I'm not really sure why my code has all three, and yours doesn't.  For that matter.. my .html file doesn't actually render, it just displays the html markup.. so I guess I'm just generally not sure what's happening.

But still, I want to see your plot!

**Comment by aaronmck** [comment]
It's kind of weird to have anything but the Rmd file checked in, since it generates both the html file and the .md file (plain markdown).  Might be a good question for tomorrow in class.

**Comment by bvernot** [comment]
Yeah, that's a good point.  I think that since the .mb file is rendered,
it's useful to commit.  The html file doesn't make as much sense to me.
